SUPPORTED_APIS.md

```diff
@@ -105,7 +105,7 @@ The `status` options are:
 | `HARDHAT` | `hardhat_setCoinbase` | `NOT IMPLEMENTED` | Sets the coinbase address |
 | `HARDHAT` | `hardhat_setLoggingEnabled` | `NOT IMPLEMENTED` | Enables or disables logging in Hardhat Network |
 | `HARDHAT` | `hardhat_setMinGasPrice` | `NOT IMPLEMENTED` | Sets the minimum gas price |
-| `HARDHAT` | `hardhat_setNextBlockBaseFeePerGas` | `NOT IMPLEMENTED` | Sets the base fee per gas for the next block |
+| [`HARDHAT`](#hardhat-namespace) | [`hardhat_setNextBlockBaseFeePerGas`](#hardhat_setnextblockbasefeepergas) | `SUPPORTED` | Sets the base fee per gas for the next block |
```

Comment:
why did you add `(#hardhat_setnextblockbasefeepergas)`?
it is needed only if there is a description here - https://github.com/vbar/era-test-node/blob/set-next-block-base-fee-per-gas/SUPPORTED_APIS.md#network-namespace.

Reply:
Just because all the other lines had it - I didn't realize it needs to have a target written as well... I've removed the link.

Reply:
We need to add this back with a section dedicated to explaining what this endpoint is and an example curl request

Reply:
The reason I didn't do that is that those sections start with a source link, and I wasn't sure where that would go - just https://github.com/vbar/era-test-node/blob/main/src/node/hardhat.rs ?

Reply:
Yes, wherever the logic is.

Reply:
OK, added section.